ext/crypto/00_crypto.js

```diff
@@ -74,6 +74,7 @@
 
   const supportedAlgorithms = {
     "digest": {
+      "MD-5": null,
```

Comment:
Is this "safe"? It will be possible to do this now and algorithm normalization won't complain, it will error at some later step: 
```rust
{
  name: "RSASSA-PKCS1-v1_5",
  modulusLength: 2048,
  publicExponent: new Uint8Array([0x01, 0x00, 0x01]), 
  hash: {
    name: "MD5"
  }
};
```

Reply:
The thought has occurred to me but I didn't see a good alternative. How would you do it?

Reply:
I can't think of a clean way but we can maybe create a special path for MD5 in `digest()` itself, instead of doing WebCrypto normalization.

Reply:
Not a bad idea, PTAL.